### DISCUSSION TITLE
Additional Moderators

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-pkgs#25830] [General] [answered]
We've been having some discussion deep in the bottom of a [PR](https://github.com/microsoft/winget-pkgs/pull/24815#issuecomment-901302890) about adding additional moderators. 

@ItzLevvie, @jedieaston, and @OfficialEsco have all had the opportunity to share suggestions.

We need to draft a document for what is expected of moderators and a standard process for how PRs are reviewed. We hope this will help with consistency, and continue improving the overall quality of manifests being submitted.

@ImJoakim, @KaranKad, and @quhxl have all been identified as candidates, and they have all expressed willingness to help out.

Once we get the document on moderation merged, we will make sure everyone still agrees to the extra expectations, and then we will start adding new moderators.

**Comment by denelon** [comment]
https://github.com/microsoft/winget-pkgs/pull/25831

**Reply by vedantmgoyal9** [reply]
@denelon I am also interested in being a moderator.

**Reply by denelon** [reply]
@vedantmgoyal2009, thanks for letting me know. I've observed many of your PRs and interaction with the community. Keep up the good work, and keep learning. I've already seen improvements from your contributions based on feedback from others. When the other moderators and the team here at Microsoft observe consistent high quality work and have no concerns for areas of improvement, you are more likely be nominated. Another subtle note to be aware of is the other moderators didn't ask to be nominated. Their work "spoke" for itself. It can be hard to navigate recognition in open source. Be assured, we are aware of the work you are doing, and it is highly valued.

**Comment by denelon** [comment, marked as answer]
Congratulations @ImJoakim, @KaranKad, and @quhxl. You have all been made moderators.

**Comment by denelon** [comment]
Hello everyone, I've received multiple nominations for @Trenly to become a moderator.

@Trenly if you agree with the guidelines for [moderation](https://github.com/microsoft/winget-pkgs/blob/master/Moderation.md), and you are willing to serve as a community moderator, please reply to this comment. There is no pressure here, this is an entirely voluntary role.

**Reply by Trenly** [reply]
I'd be honored to!
